src/lamp_py/tableau/jobs/filtered_hyper.py

```diff
@@ -65,7 +65,7 @@ def create_tableau_parquet(self, num_days: Optional[int]) -> bool:
         # previous days uploads working
         # need to implement new input daily_upload_hour as well
         # this will currently update hourly. will monitor
-        end_date = datetime.now()
+        end_date = datetime.now().date()
```

Comment:
You've got the `end_date` param of `file_list_from_s3_date_range` as accepting `datetime.datetime` type, not `datetime.date`